log4j-core/src/main/java/org/apache/logging/log4j/core/net/SmtpManager.java

```diff
@@ -16,24 +16,16 @@
  */
 package org.apache.logging.log4j.core.net;
 
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
-import java.io.OutputStream;
-import java.util.Date;
-import java.util.Properties;
+import java.io.*;
```

Comment:
You'll want to change your formatter import settings to not collapse them to `*` except for static imports.